src/vconsole/vconsole-setup.c

```diff
@@ -134,7 +130,7 @@ static int toggle_utf8_sysfs(bool utf8) {
 
         r = write_string_file("/sys/module/vt/parameters/default_utf8", utf8 ? "1" : "0", 0);
         if (r < 0)
-                log_warning_errno(r, "Failed to %s sysfs UTF-8 flag: %m", utf8 ? "enable" : "disable");
+                (void) log_warning_errno(r, "Failed to %s sysfs UTF-8 flag: %m", utf8 ? "enable" : "disable");
```

Comment:
(void) is not needed here. (void) is only used for functions which are usually checked.